catalog/core/catalog-core-api-impl/pom.xml

```diff
@@ -49,6 +49,18 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Embed-Dependency>
+                            platform-util
+                        </Embed-Dependency>
+                    </instructions>
+                </configuration>
+            </plugin>
```

Comment:
You shouldn't need to embed this since the impl itself is a jar. Embedding this won't allow other bundles to take advantage of it, they just also need to embed platform-util themselves.